akka-cluster/src/main/scala/akka/cluster/Member.scala

```diff
@@ -32,7 +33,11 @@ class Member private[cluster] (
     case m: Member ⇒ uniqueAddress == m.uniqueAddress
     case _         ⇒ false
   }
-  override def toString = s"Member(address = ${address}, status = ${status})"
+  override def toString =
+    if (team == ClusterSettings.DefaultTeam)
+      s"Member(address = $address, status = $status)"
+    else
+      s"Member(address = $address, team = $team, status = $status)"
 
   def hasRole(role: String): Boolean = roles.contains(role)
 
```

Comment:
Something is needed when merging Gossip also. When a member is removed from one team it should remain removed after merging with a Gossip in another team. I think we can use the fact that a member can only be removed by leader in own team. See `Member.pickHighestPriority` and `GossipSpec`.